components/requests/DoctorInformationCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, HStack, Text, Divider, SimpleGrid, Link } from '@chakra-ui/react';
+import Card from '@components/internal/Card';
+
+type doctorInformationProps = {
+  readonly name: string;
+  readonly mspNumber: number;
+  readonly phoneNumber: string;
+  readonly province: string;
+  readonly city: string;
+  readonly address: string;
+  readonly postalCode: string;
+};
+
+export default function DoctorInformationCard(props: doctorInformationProps) {
+  return (
+    <Card w="738px" h="346px">
```

Comment:
We don't want to use hardcoded values for the width and height since we have t he 12-column grid layout. It's kind of hard to find in the Figma, but if you go to RCD Experience > Hifi Screens > Requests and toggle on Layout Grids using Ctrl+Shift+4, you will see the layout grid on this screen. To take advantage of this, please use the `GridItem` component, which you can see in examples on other pages of our app.

Reply:
Good call! Thanks for the suggestions